scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -188,7 +188,7 @@ def main(vds, ancestry_file_location, sites_only_vcf_path, dry_run_n_parts=None)
         parts_per_round = dry_run_n_parts
         ht_paths = [sites_only_vcf_path.replace(r".sites-only.vcf.bgz", f'_dryrun.ht')]
         sites_only_vcf_path = sites_only_vcf_path.replace(r".vcf.bgz", f'_dryrun.vcf.bgz')
-    else:
+    elif false:
```

Comment:
This isn't right. You just want to change the for loop below to `for i in []:` to skip to the merge step

Reply:
omg embarrassing! I totally missed the lack of an indent--thank you!!!